### PR TITLE
prepare release for resourcemanager

### PR DESCRIPTION
**Pull request by ArcturusZhang**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

File: sdk/resourcemanager/Azure.ResourceManager/CHANGELOG.md [hunk 1]
```diff
@@ -1,14 +1,22 @@
...
 
-### Breaking Changes
+- New struct `ArmEnvironment`.
 
-### Bugs Fixed
+### Breaking Changes
 
-### Other Changes
+- Now all the resource classes will have a `Resource` suffix (if it previously does not have one).
+- Rename some model to a more comprehensive name.
```
Comment (archerzz) [inline sdk/resourcemanager/Azure.ResourceManager/CHANGELOG.md:12]:
```suggestion
- Rename some models to more comprehensive names.
```

File: sdk/resourcemanager/Azure.ResourceManager/CHANGELOG.md [hunk 2]
```diff
@@ -1,14 +1,22 @@
...
+- Class `ManagementGroupResource` (previously `ManagementGroup`), `ManagementGroupCollection` and `ManagementGroupData` move from `Azure.ResourceManager.Management` namespace to `Azure.ResourceManager.ManagementGroups`.
```
Comment (archerzz) [inline sdk/resourcemanager/Azure.ResourceManager/CHANGELOG.md:13]:
```suggestion
- Class `ManagementGroupResource` (previously `ManagementGroup`), `ManagementGroupCollection` and `ManagementGroupData` are moved from `Azure.ResourceManager.Management` namespace to `Azure.ResourceManager.ManagementGroups`.
```

File: sdk/resourcemanager/Azure.ResourceManager/CHANGELOG.md [hunk 3]
```diff
@@ -1,14 +1,22 @@
...
+- Class `ArmResource` and `ArmCollection` move from `Azure.ResourceManager.Core` to `Azure.ResourceManager`.
+- Namespace `Azure.ResourceManager.Core` and `Azure.ResourceManager.Management` are removed.
+- Class `ErrorDetail` and `ErrorAdditionalInfo` are removed.
```
Comment (archerzz) [inline sdk/resourcemanager/Azure.ResourceManager/CHANGELOG.md:16]:
```suggestion
- Remove class `ErrorDetail` and `ErrorAdditionalInfo`.
```

File: sdk/resourcemanager/Azure.ResourceManager/CHANGELOG.md [hunk 4]
```diff
@@ -1,14 +1,22 @@
...
+- New struct `ArmEnvironment`.
 
-### Bugs Fixed
+### Breaking Changes
 
-### Other Changes
+- Now all the resource classes will have a `Resource` suffix (if it previously does not have one).
+- Rename some model to a more comprehensive name.
+- Class `ManagementGroupResource` (previously `ManagementGroup`), `ManagementGroupCollection` and `ManagementGroupData` move from `Azure.ResourceManager.Management` namespace to `Azure.ResourceManager.ManagementGroups`.
+- Class `ArmResource` and `ArmCollection` move from `Azure.ResourceManager.Core` to `Azure.ResourceManager`.
```
Comment (archerzz) [inline sdk/resourcemanager/Azure.ResourceManager/CHANGELOG.md:14]:
```suggestion
- Move class `ArmResource` and `ArmCollection` from `Azure.ResourceManager.Core` to `Azure.ResourceManager`.
```

File: sdk/resourcemanager/Azure.ResourceManager/CHANGELOG.md [hunk 5]
```diff
@@ -1,14 +1,22 @@
...
+- Namespace `Azure.ResourceManager.Core` and `Azure.ResourceManager.Management` are removed.
```
Comment (archerzz) [inline sdk/resourcemanager/Azure.ResourceManager/CHANGELOG.md:15]:
```suggestion
- Remove namespace `Azure.ResourceManager.Core` and `Azure.ResourceManager.Management`.
```

File: sdk/resourcemanager/Azure.ResourceManager/CHANGELOG.md [hunk 6]
```diff
@@ -1,14 +1,22 @@
...
-## 1.0.0-beta.9 (Unreleased)
+## 1.0.0-beta.9 (2022-03-31)
 
 ### Features Added
 
-### Breaking Changes
+- New struct `ArmEnvironment`.
 
-### Bugs Fixed
+### Breaking Changes
```
Comment (Yao725) [inline sdk/resourcemanager/Azure.ResourceManager/CHANGELOG.md:9]: It looks like that we missed the change `Objetc` to `BinaryData`

File: sdk/resourcemanager/Azure.ResourceManager/CHANGELOG.md [hunk 7]
```diff
@@ -1,14 +1,22 @@
...
 
-### Other Changes
+- Now all the resource classes will have a `Resource` suffix (if it previously does not have one).
+- Rename some models to more comprehensive names.
```
Comment (Yao725) [inline sdk/resourcemanager/Azure.ResourceManager/CHANGELOG.md:12]: Maybe `Renamed` is better? Because the previous changelogs all used the past tense.

Comment (ArcturusZhang) [reply sdk/resourcemanager/Azure.ResourceManager/CHANGELOG.md:12]: I jsut checked the changelog of azure.core. They are using past tense.
I will change the tense here

**Review by Yao725 — APPROVED** [review]

**Review by ArthurMa1978 — APPROVED** [review]

**Pull request merged**